### PR TITLE
GQL-52: Allow querying for Acls from within a group

### PR DESCRIPTION
**Pull request by abbottry**
# Overview

### What is the feature?

Adds the ability to query for Acls that belong to a group, from within a group query.

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/resolvers/group.js [hunk 1]
```diff
@@ -79,6 +80,23 @@ export default {
...
         requestInfo
       )
+    },
+
+    acls: async (source, args, context, info) => {
+      const { groupId } = source
+
+      const { dataSources } = context
+
+      const a = await dataSources.aclSourceFetch(
```
Comment (dmistry1) [inline src/resolvers/group.js:90]: Here I think we can just return the dataSource, don't think a `const` is needed. 
 ```
return dataSources.aclSourceFetch(
   handlePagingParams({
     ...args,
     permittedGroup: groupId
   }),
   context,
   parseResolveInfo(info)
 ) 
```

Comment (abbottry) [reply src/resolvers/group.js:90]: Oh man, good catch, I was testing something.

**Review by dmistry1 — APPROVED** [review]

File: src/resolvers/acl.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ export default {
     },
 
     catalogItemIdentity: async (source) => {
-      const { catalogItemIdentity } = source
+      const { catalogItemIdentity = {} } = source
```
Comment (macrouch) [inline src/resolvers/acl.js:140]: Do you have to worry about null here?

Comment (abbottry) [reply src/resolvers/acl.js:140]: No sir, we're good.

**Review by macrouch — APPROVED** [review]

**Pull request merged**